Library/Formula/tidy-html5.rb

```diff
@@ -0,0 +1,14 @@
+class TidyHtml5 < Formula
+  homepage "http://www.html-tidy.org/"
+  url "https://github.com/htacg/tidy-html5/archive/tag-4.9.25.tar.gz"
+
+  depends_on "cmake" => :build
+
+  def install
+    ENV.deparallelize
+    cd "build/cmake"
+    system "cmake", "../..", "-DCMAKE_INSTALL_PREFIX=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!